hazelcast/src/test/resources/hazelcast-fullconfig-without-network.yaml

```diff
@@ -64,7 +64,6 @@ hazelcast:
             consistency-check-strategy: MERKLE_TREES
 
         istanbul:
-          class-name: com.hazelcast.wan.custom.WanPublisher
```

Comment:
Why removing this?

Reply:
This property [is not parsed](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/internal/config/MemberDomConfigProcessor.java#L689) so I didn't schematize it and removed from the example as well.

Reply:
Right. It's not anymore used indeed. We have this node for the `custom-publisher` only, which is not at all in the fullconfig... We have it in `hazelcast-full-example.yaml`, so if it's tested in this PR, we're fine.